docs/sources/user_guide/runtime_assurance.rst

```diff
@@ -216,3 +217,21 @@ The mathematical formulations used by the RAM rely on an implicit assumption abo
   There exist a fundamental trade-off between conservatism and optimality at the control level. The more certainty one has in things like dynamics, localization, and communication timing, the less conservatism must be introduced to ensure a low probability of collision.
 
   The Supervisor is able to account for uncertainty in an explicit and quantitative way. Please `contact 3laws`_ to learn more about the process of tailoring Supervisor to better handle uncertainties and communication delays in your system.
+
+
+Visualization
+=============
+
+The RAM publishes a set of markers to visualize the processing done by the Safety Maps, and the safety constraints generated by the Kernel Generator:
+
+.. image:: ../data/ram_viz.png
+  :width: 800px
+  :align: center
+  :alt: RAM visualization
+
+|
+
+This data is published on the **/lll/ram/markers** topic. The :ref:`control panel <control_panel_viz>` can provide you with a an rviz configuration file to visualize this data.
+
+.. important::
+  If the **Use Localization** setting is set to false, the robot will stay still at the center of the world, and the raw laserscan data likely won't line up with the Safety Map markers.
```

Comment:
Would it be more precise to say "the robot's representation will stay still...", because the robot is in motion, but from the robot's perspective, the world is moving relative to it.  Also, why would the raw laserscans not line up with the Safety Map markers?  Aren't the markers derived from the raw laserscans, independently of localization?

Reply:
Localization of raw laserscan is driven by the frame_id field in the message, for which we have no control over. If people have issues, we'll elaborate.

Reply:
I've updated to suggest using the robot base frame as world frame parameter to get an consistent viz